modules/benchmarks/FilterBenchmark.scala.output.md

```diff
@@ -0,0 +1,67 @@
+# The configuration and hardware info
+
+```
+Software:
+
+  System Version: macOS 13.2.1 (22D68)
+  Kernel Version: Darwin 22.3.0
+  Boot Volume: Macintosh HD
+  Boot Mode: Normal
+  Computer Name: MacBook Pro (21)
+  Secure Virtual Memory: Enabled
+
+Hardware:
+
+  Model Name: MacBook Pro
+  Model Identifier: MacBookPro18,1
+  Model Number: Z14Y00047DK/A
+  Chip: Apple M1 Pro
+  Total Number of Cores: 10 (8 performance and 2 efficiency)
+  Memory: 32 GB
+```
+
+```
+[info] # JMH version: 1.32
+[info] # VM version: JDK 17.0.1, OpenJDK 64-Bit Server VM, 17.0.1+12
+[info] # VM invoker: /Users/dkFePaHa/.asdf/installs/java/adoptopenjdk-17.0.1+12/bin/java
```

Comment:
Do we want these locations to be in the code?

Reply:
I guess all the neccessary info is in the line above – I'll remove it